definitions/mobile.d.ts

```diff
@@ -6,49 +6,92 @@ declare module Mobile {
 		skipRefresh?: boolean;
 	}
 
+	interface IDeviceInfo {
+		identifier: string;
+		displayName: string;
+		model: string;
+		version: string;
+		vendor: string;
+		platform: string;
+	}
+	
 	interface IDevice {
-		getIdentifier(): string;
-		getInstalledApplications(): IFuture<string[]>;
-		getDisplayName(): string;
-		getModel(): string;
-		getVersion(): string;
-		getVendor(): string;
-		getPlatform(): string;
+		deviceInfo: Mobile.IDeviceInfo;
+		applicationManager: Mobile.IDeviceApplicationManager;
+		fileSystem: Mobile.IDeviceFileSystem;
 		deploy(packageFile: string, packageName: string): IFuture<void>;
-		sync(localToDevicePaths: ILocalToDevicePathData[], appIdentifier: IAppIdentifier, liveSyncUrl: string): IFuture<void>;
-		sync(localToDevicePaths: ILocalToDevicePathData[], appIdentifier: IAppIdentifier, liveSyncUrl: string, options: ISyncOptions): IFuture<void>;
 		openDeviceLogStream(): void;
-		runApplication(applicationId: string): IFuture<void>;
-		uninstallApplication(applicationId: string): IFuture<void>;
-		listFiles(devicePath: string): IFuture<void>;
-		getFile(deviceFilePath: string): IFuture<void>;
-		putFile(localFilePath: string, deviceFilePath: string): IFuture<void>;
+	}
+	
+	interface IAndroidDevice extends IDevice {
+		adb: Mobile.IAndroidDebugBridge;		
+		sendBroadcastToDevice(action: string, extras: IStringDictionary): IFuture<number>;
 	}
 
-	interface IAppIdentifier {
+	interface IIOSDevice extends IDevice {
+		startService(serviceName: string): number;
+		mountImage(): IFuture<void>;
+		tryToExecuteFunction<TResult>(func: () => TResult): TResult;
+	}
+	
+	interface IDeviceAppDataBase {
 		appIdentifier: string;
-		deviceProjectPath: string;
-		liveSyncFormat: string;
-		encodeLiveSyncHostUri(hostUri: string): string;
-		isLiveSyncSupported(device: any): IFuture<boolean>;
-		getLiveSyncNotSupportedError(device: any): string;
 	}
 
-	interface IAndroidDevice extends IDevice {
-		debug(packageFile: string, packageName: string, debuggerSetup?: any): IFuture<void>;
+	interface IDeviceAppData extends IDeviceAppDataBase {
```

Comment:
Consider removing `Data` suffix from the name here because this is not DTO. It has a method: `isLiveSyncSupported(...)`